### PR TITLE
Feature/ju 13 java 21 migration fearghus

### PR DESCRIPTION
**Pull request by foman-ch**
**Jira**: https://companieshouse.atlassian.net/browse/JU-128

### CODE REVIEW

**Review by hwatkins-ch — COMMENTED** [review]
Looks good, a few dependencies need upgrading still, and just a question around the addition of the environment variable, do we need to check this needs to be added?

File: src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/environment/RequiredEnvironmentVariables.java [hunk 1]
```diff
@@ -6,9 +6,10 @@ public enum RequiredEnvironmentVariables {
     MONGODB_URL("MONGODB_URL"),
     CHS_API_KEY("CHS_API_KEY"),
     API_URL("API_URL"),
-    PAYMENTS_API_URL("PAYMENTS_API_URL");
+    PAYMENTS_API_URL("PAYMENTS_API_URL"),
+    DOCUMENT_API_URL("DOCUMENT_API_URL");
```
[the remark this answers is not in the document]
Comment (foman-ch) [reply src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/environment/RequiredEnvironmentVariables.java]: Addition of startup variable in tilt was enough - thanks

**Comment by foman-ch** [comment]
> Looks good, a few dependencies need upgrading still, and just a question around the addition of the environment variable, do we need to check this needs to be added?

Sorted, thanks for the review!

**Review by hwatkins-ch — APPROVED** [review]

File: suppress.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vshastri-ch) [inline suppress.xml:1]: Would like to understand if this is essential

**Review by akunati-ch — APPROVED** [review]
Looks good

**Pull request merged**